packages/core/src/customizations/src/GlobalOverrides.tsx

```diff
@@ -1,9 +1,15 @@
 import WebFontsOverrides from 'src/customizations/src/components/overrides/WebFonts'
 import { default as CoreWebFonts } from 'src/fonts/WebFonts'
 import ThirdPartyScriptsOverrides from 'src/customizations/src/components/overrides/ThirdPartyScripts'
+import ThirdPartyScriptsPluginsOverrides from 'src/plugins/overrides/ThirdPartyScripts'
```

Comment:
Does B2B have any specific needs for Third Party Scripts or Web Fonts? Or has this just been added for consistency/completeness? If it's just for completeness, I'd remove it.

Reply:
We need the WebFonts, but Third-Party Scripts are not required yet, so they have been removed.